src/main/java/pl/betoncraft/betonquest/compatibility/protocollib/wrappers/UncheckedPacketException.java

```diff
@@ -0,0 +1,22 @@
+package pl.betoncraft.betonquest.compatibility.protocollib.wrappers;
+
+public class UncheckedPacketException extends RuntimeException {
+
+    private static final long serialVersionUID = 60789910395201791L;
```

Comment:
I don't think this is needed here.

Reply:
Any class that implements Serializable should haven a serialVersionUID. As the Throwable class implements this interface every exception should have this.

Reply:
that is fine, it is like it is in every other exception class from BQ